src/main/java/com/gworks/ui/controller/EdgeWebDriverController.java

```diff
@@ -0,0 +1,11 @@
+package com.gworks.ui.controller;
+
+import com.gworks.ui.services.EdgeWebDriverService;
+import org.openqa.selenium.WebDriver;
+
+public class EdgeWebDriverController {
+
+    private final EdgeWebDriverService edgeWebDriverService = new EdgeWebDriverService();
```

Comment:
I would like to add an _ (underscore) before the variable when we are creating an instance.
Just to make it neater.